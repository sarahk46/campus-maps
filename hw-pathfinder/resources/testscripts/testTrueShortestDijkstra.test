# Find a path with Dijkstra's algorithm when
# there are multiple edges and two paths to explore.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph A C 11.0
AddEdge simpleGraph B C 2.0

FindPath simpleGraph A C

