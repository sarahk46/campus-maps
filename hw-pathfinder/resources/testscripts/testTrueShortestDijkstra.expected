# Find a path with Dijkstra's algorithm when
# there are multiple edges and two paths to explore.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 11.000 from A to C in simpleGraph
added edge 2.000 from B to C in simpleGraph

path from A to C:
A to C with weight 11.000
total cost: 11.000